main.py

```diff
@@ -106,5 +110,8 @@ def on_gui_quit():
 
 
 if __name__ == "__main__":
-    multiprocessing.freeze_support()
-    main()
+    handle = CreateMutex(None, 1, "Mimic Mutex")
+    
+    if GetLastError() != ERROR_ALREADY_EXISTS:
+        multiprocessing.freeze_support()
```

Comment:
So I don't understand why, but if this line isn't directly after `if __name__ == "__main__"`, the subprocess that runs the webserver will not be started.

Reply:
Can you check if my commit that just moves it fixes the issue?

Reply:
Seems to be working now, not sure why it needs to be called so early on. 👍🏻